concent_api/core/models.py

```diff
@@ -287,24 +290,37 @@ class Meta:
             ('requestor', 'subtask_id'),
         )
 
-    task_id                     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH)
+    task_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
 
     # Golem clients are not guaranteed to use unique subtask_id because they are UUIDs,
     # but Concent at this moment does not support subtasks with non-unique IDs.
     # However, the combination of requestor's public key and subtask ID is guaranteed to be unique.
-    subtask_id                  = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, unique = True, db_index = True)
+    subtask_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH, unique=True, db_index=True)
 
     # Relation to concent client in requestor context.
-    requestor                   = ForeignKey(Client, related_name = 'subtasks_as_requestor')
+    requestor = ForeignKey(Client, related_name='subtasks_as_requestor')
 
     # Relation to concent client in provider context.
-    provider                    = ForeignKey(Client, related_name = 'subtasks_as_provider')
+    provider = ForeignKey(Client, related_name='subtasks_as_provider')
 
-    state                       = CharField(max_length = 32, choices = SubtaskState.choices())
+    state = CharField(max_length=32, choices=SubtaskState.choices())
 
     # If in an active state, it's the time at which Concent automatically transitions to a passive state if it does
     # not get the information it expects from the client (a timeout). Must be NULL in a passive state.
-    next_deadline               = DateTimeField(blank = True, null = True)
+    next_deadline = DateTimeField(blank=True, null=True)
+
+    created_at = DateTimeField(editable=False, default=timezone.now)
+
```

Comment:
I would not separate those two with an extra blank line, those are a kind of logical group.

Reply:
Merged, Done